Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/ReadMe.md

```diff
@@ -0,0 +1,16 @@
+HoloToolkit Examples - Spatial Understanding - Feature Overview
+============
+
+## Description
+The HoloToolkit.SpatialUnderstanding library encapsulates this technology, allowing you to quickly find empty spaces on the walls, place objects on the ceiling, identify placed for character to sit, and a myriad of other spatial understanding queries.
+
+There are three primary interfaces exposed by the module: topology for simple surface and spatial queries, shape for object detection, and the object placement solver for constraint based placement of object sets. 
+
+In addition to the three primary module interfaces, a ray casting interface can be used to retrieve tagged surface types and a custom watertight playspace mesh can be copied out.
+
+This sample demonstrates many of the features of the Spatial Understanding addon. 
+
+## SETUP
+1. Import the HoloToolkit into the project
+2. Open the Scenes\Main level
+3. Build for HoloLens
```

Comment:
I know "Build for HoloLens" step should technically cover this, but you might want to add "Don't forget to turn on the SpatialPerception capability under Player settings." I created a new Unity project, used the HoloToolkit menu item to apply project settings, then imported your scene. When I first ran it on the device it took a minute or two before dawned on me why it wouldn't progress beyond the "Scan your play space" message. Doh!